### PR TITLE
Added a click event loop to all cells

### PR DESCRIPTION
**Pull request by yohanology**
Created a loop to apply click event function to all cells but it's not working.

### CODE REVIEW

**Comment by yohanology** [comment]
I fixed the loop problem. Replaced it with a specific dom selection with parent() and index() to apply the click events on the cells.

File: index.html [hunk 1]
```diff
@@ -16,6 +16,56 @@
     <script type="text/javascript" src="js/application.js"></script>
```
Comment (h8rry) [inline index.html:16]: You still have indentation problems in this file

File: index.html [hunk 2]
```diff
@@ -16,6 +16,56 @@
...
+    <div class="container">
+        <div class="row">
+                <h1>Minesweeper Reborn</h1>
+                <h3>Avoid the bomb and flag them or you'll get blown into pieces.</h3>
+        </div>
+        <div>
+            <table id="open-board" class="table table-bordered">
+               <tbody id="tableBody">
+                    <tr>
+                        <td></td>
```
Comment (h8rry) [inline index.html:28]: Do you need these empty `<td>`?

**Comment by h8rry** [comment]
I shouldn't be able to click on cells that have already been expanded. Otherwise, you will double-count the bomb if I click twice and die early.

File: index.html (hunk 2 above)
Comment (h8rry) [inline index.html:28]: This works, but if you want a bigger mine or a dynamic size mine, you should generate this dyanmically

File: js/application.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      if (openBoard[row][col] === 'X') {
+        console.log('Game over. You just blew up into pieces.')
+      }
+      if (openBoard[row][col] === 0) {
+            openAround(row,col);
+      }
+    
+  }
+  
+  //Open cells around a cell
```
Comment (h8rry) [inline js/application.js:21]: Please explain the logic to me after the presentation

**Comment by h8rry** [comment]
Looks good

**Pull request merged**